### PR TITLE
v1.0 of Pina-Golada

### PR DESCRIPTION
**Pull request by LynxPlay101**
This is version 1.0 of Pina-Golada have fun

### CODE REVIEW

**Comment by HeavyWombat** [comment]
@LynxPlay101 I did a first pass over the files. Since there are a lot new files, it will take a bit of time.

One thing that struck me already: Is it possible that you switched the IDE and this comes with different preferences regarding comments? It seems that some existing comments were changed from `//` to `/* ... */` style. Also, there are instances of `_ := It("...")`, where the `_ :=` usually is omitted for better readability. Can you have a look into your preferences and change it the the common style used by `disrupt-o-meter`? Thanks in advance!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	The asset path passed to the annotation is relative to the location the tool will be called in.
+	The compressor defines what type of compression will be used to store the asset found as binary.
+	@pgl(asset=/assets/default-config.yaml&compressor=tar)
+	*/
+	GetAssetFile() (dir files.Directory , e error)
+	
+}
+```
+
+Now run `pgl generate` prior to your build. This will generate all needed files.
```
Comment (HeavyWombat) [inline README.md]: I think the tool is called `pina-golada` in the build script. We have to adjust the documentation here.

File: cmd/golada/main.go [hunk 2]
```diff
@@ -1,22 +1,22 @@
-// Copyright © 2019 The Homeport Team
+//Copyright © 2019 The Homeport Team
```
Comment (HeavyWombat) [inline cmd/golada/main.go]: Any chance to get GoLand to give me that beloved space after the double slash?

File: scripts/test.sh [hunk 3]
```diff
@@ -22,7 +22,7 @@
 
 set -euo pipefail
 
-GO111MODULE=on ginkgo -r \
+GO111MODULE=on ginkgo ./... -r \
```
Comment (HeavyWombat) [inline scripts/test.sh]: Out of curiosity, was that added by GoLand?

Comment (LynxPlay101) [reply scripts/test.sh]: No I added it, as for whatever reason ginkgo kept failing without a specified path

File: cmd/golada/main.go (hunk 2 above)
Comment (LynxPlay101) [reply cmd/golada/main.go]: GoLand can do whatever your heart desiers ;)

File: scripts/test.sh (hunk 3 above)
Comment (LynxPlay101) [reply scripts/test.sh]: Works now, removed it again

**Comment by LynxPlay101** [comment]
@HeavyWombat the cocktail is ready to be served to the master branch 👌

**Pull request merged**